helpers/cuentasContablesHelper/cuentasContables.helper.go

```diff
@@ -235,24 +250,9 @@ func AsientoContable(totales map[int]float64, tipomvto string, descripcionMovto
 					transaccion.Movimientos = append(transaccion.Movimientos, movimientoCredito)
 				}
 			}
-
-			apiMvtoContables := "http://" + beego.AppConfig.String("movimientosContablesmidService") + "transaccion_movimientos/transaccion_movimientos/"
-			outputError = map[string]interface{}{"funcion": "asientoContable - en desarrollo", "status": "500", "err": err}
-
-			if err := request.SendJson(apiMvtoContables, "POST", &res, &transaccion); err != nil {
-				logs.Error(err)
-				outputError = map[string]interface{}{
-					"funcion": "AsientoContable -if err := request.SendJson(apiMvtoContables, \"POST\", &res, &transaccion);",
-					"err":     err,
-					"status":  "502",
-				}
-			} else {
-				logs.Info("Transaccion contable")
-				logs.Info("******** correcto *********")
-				res["resultadoTransaccion"] = transaccion
-				return res, nil
-			}
 		} else {
+
+			logs.Debug("******* no encontro las cuentas")
```

Comment:
Comentar
```suggestion
			// logs.Debug("******* no encontro las cuentas")
```

o Eliminar
```suggestion
```